spec/controllers/organizations_controller_spec.rb

```diff
@@ -285,11 +285,22 @@
   end
 
   describe "PATCH #update", :vcr do
-    it "correctly updates the organization" do
-      options = { title: "New Title" }
-      patch :update, params: { id: organization.slug, organization: options }
+    context "when success" do
+      it "correctly updates the organization" do
+        options = { title: "New Title" }
+        patch :update, params: { id: organization.slug, organization: options }
 
-      expect(response).to redirect_to(organization_path(Organization.find(organization.id)))
+        expect(response).to redirect_to(organization_path(Organization.find(organization.id)))
+      end
+    end
+
+    context "when fail" do
+      it "not change current organization title" do
+        options = { title: " " }
+        patch :update, params: { id: organization.slug, organization: options }
+
+        expect(assigns(:current_organization).title).not_to eql(options[:title])
```

Comment:
Can you add one final assertion in here showing we redirect to :edit? Then I think this will be good to go :sparkles:

Reply:
all done